### PR TITLE
DOC-2513: New `importword_converter_options` option that allows customizing the conversion.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2513

Site: [Release notes](http://docs-feature-74-doc-2513tiny-11219.staging.tiny.cloud/docs/tinymce/latest/7.4-release-notes/#new-importword_converter_options-option-that-allows-customizing-the-conversion)
Site: [Included previously removed API ref link](http://docs-feature-74-doc-2513tiny-11219.staging.tiny.cloud/docs/tinymce/latest/importword/#api-reference)
Site: [importword-converter-options](http://docs-feature-74-doc-2513tiny-11219.staging.tiny.cloud/docs/tinymce/latest/importword/#importword-converter-options)

Changes:
* added new option to `release-notes.adoc`
* added new option to `importword.adoc`
* added api-reference link (previously removed) back to `importword.adoc` file.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`.
- [x] Included a `release note` entry for any `New product features`.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/importword.adoc [hunk 1]
```diff
@@ -24,4 +24,93 @@ tinymce.init({
...
+  // ... other options ...
+  importword_converter_options: {
+    formatting: {
+      styles: 'none'
+    }
+  }
+});
+----
+
+**Effect:** Setting this to `'none'` will exclude Word styles from the generated document.
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/importword.adoc:116]: @ShiridiGandham Lets discuss if this is required in the `<plugin-page>.adoc` or if the api ref docs is sufficient.

Comment (kemister85) [reply modules/ROOT/partials/configuration/importword.adoc:116]: We have decided to remove this section from the plugin docs and rely on the REST API docs for this.

File: modules/ROOT/partials/configuration/importword.adoc [hunk 2]
```diff
@@ -24,4 +24,93 @@ tinymce.init({
...
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/importword.adoc:116]:
```suggestion
```

File: modules/ROOT/partials/configuration/importword.adoc [hunk 3]
```diff
@@ -24,4 +24,28 @@ tinymce.init({
...
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'importword',
+  toolbar: 'importword',
+  importword_converter_options: {
+    formatting: {
+      resets: 'inline',
+      defaults: 'inline',
+      styles: 'none',
+    }
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/importword.adoc:48]:
```suggestion
    formatting: {
      resets: 'inline', // Minimize differences between Word and HTML default styles.
      defaults: 'inline', // Control inclusion of default formatting.
      styles: 'none', // Manage Word style inclusion in the generated document.
    }
```

Comment (kemister85) [reply modules/ROOT/partials/configuration/importword.adoc:48]: @FarzadHayat would you agree with this update?

**Review by ztomaszyk — APPROVED** [review]

**Pull request merged**